mobile/src/bundles/common/enums/ui/colors.enum.ts

```diff
@@ -1,6 +1,6 @@
 const Color = {
     PRIMARY: '#274F8D',
-    SECONDARY: '#18A0FB',
+    SECONDARY: '#2E61AE',
```

Comment:
```suggestion
```

Reply:
i guess it's shall fail some styles, didnt check other pr's, atleast btns styles will fail, delete it and fix issues?

Reply:
Yes